src/rust/src/conversion_s_to_r.rs

```diff
@@ -49,6 +49,7 @@ pub fn pl_series_to_list(
                         .collect_robj()
                         .set_class(&["integer64"])
                         .expect("internal error could not set class label 'integer64'")
+                        .clone()
```

Comment:
Increment reference counter to go from `&mut Robj` to `Robj` and likewise below.